### ISSUE TITLE
PLANET-4334 Allow a list of non-P4 urls not to have the ↗️ icon 

### ISSUE THREAD

**Issue by planet-4** [greenpeace/planet4#174]
We add an external icon on links that point to an external domain. As a follow up we should not display the external link icon for some links we use regularly for engagement opportunities (eg. `act.greenpeace.org`).

**Requirements**
 - Add a new field _"Commonly used external domains"_ under _Planet 4 > Social_.
 - Use a textaea field to hold multiple domains.
 - Help text: _"Links for these domains will not open in a new tab and will hide the external link icon."_
 - Both title and description should be translatable.
 - Adjust external link behaviour to remove _external link icon_ and _opening in a new tab_ for these domains.

```
Reporter: ltiralon
Sections: Styling/Theme
```

**Comment by oekeur** [comment]
Hi friends 🤗

Sure this won't interfere in a weird way if it's implemented like this?
The pre-connect domains are a technical solution for pre-connecting certain assets. If we piggyback on this by refactoring it to "Preferred external domains", that would mean that certain third party domains unintentionally might also lack the external link icon..
I know it's a bit of a edge case, but just to be sure. :)

**Comment by comzeradd** [comment]
Yeah, we discussed this also in the team. It may make more sense to have an explicit configuration option for this. I'll update the description requirements.

**Comment by oekeur** [comment]
So what shall we do for this? :)
A separate option for the 'domains without link icon'?

**Comment by comzeradd** [comment]
Description updated. Let's add a new configuration option for this.

**Comment by oekeur** [comment]
You can assign this one to me :)

**Comment by oekeur** [comment]
Whoops, sorry, this one fell off my radar!:
Starting -now really- on it now

**Comment by oekeur** [comment]
PR opened: https://github.com/greenpeace/planet4-master-theme/pull/1746
